docs/features.md

```diff
@@ -129,7 +129,7 @@
 * `ceremonyMaster` sends `ApprovedBlock` to `validatorA` and `validatorB`
 * `validatorA` and `validatorB` transition to ApprovedBlockReceivedHandler
 * `ceremonyMaster`, `validatorA` and `validatorB` tip points to block (genesis) where it has no parent and Bonds holds `validatorA` and `validatorB`
-* `readOnlyA` **never** transitions to `ApprovedBlockReceivedHandler`
+* `readOnlyA` transitions to `ApprovedBlockReceivedHandler`
```

Comment:
@rabbitonweb ^

Reply:
This is the correct outcome. readOnlyA should not be able to validate the unapproved block AND should be able to receive the approved block and transition once the network launches.